.env.example

```diff
@@ -1,45 +1,22 @@
 CELERY_TASK_ALWAYS_EAGER=True
 DJANGO_LOG_LEVEL=INFO
 LOG_LEVEL=INFO
+OPEN_DISCUSSIONS_DEFAULT_SITE_KEY=
+OPEN_DISCUSSIONS_BASE_URL=http://od.odl.local:8063
+OPEN_DISCUSSIONS_COOKIE_NAME=discussions
+OPEN_DISCUSSIONS_COOKIE_DOMAIN=odl.local
+OPEN_DISCUSSIONS_MAX_COMMENT_DEPTH=6
+OPEN_DISCUSSIONS_JWT_SECRET=
+OPEN_DISCUSSIONS_USE_S3=False
+INDEXING_API_USERNAME=mitodl
+MAILGUN_SENDER_DOMAIN=
 MAILGUN_KEY=
 MAILGUN_RECIPIENT_OVERRIDE=
 STATUS_TOKEN=
-SENTRY_LOG_LEVEL=
-SENTRY_DSN=
 UWSGI_THREAD_COUNT=5
-OPEN_DISCUSSIONS_REDDIT_CLIENT_ID=
-OPEN_DISCUSSIONS_REDDIT_SECRET=
-OPEN_DISCUSSIONS_MAX_COMMENT_DEPTH=
-OPEN_DISCUSSIONS_COOKIE_NAME=
-OPEN_DISCUSSIONS_COOKIE_DOMAIN=
-OPEN_DISCUSSIONS_JWT_SECRET=
-OPEN_DISCUSSIONS_DEFAULT_SITE_KEY=
-OPEN_DISCUSSIONS_BASE_URL=
-OPEN_DISCUSSIONS_USE_S3=False
 ELASTICSEARCH_INDEX=discussions_local
 AWS_ACCESS_KEY_ID=
 AWS_SECRET_ACCESS_KEY=
 AWS_STORAGE_BUCKET_NAME=
 CLOUDFRONT_DIST=
-INDEXING_API_USERNAME=mitodl
 SOCIAL_AUTH_MICROMASTERS_LOGIN_URL=
-
-# SAML SP
-FEATURE_SAML_AUTH=False
-SOCIAL_AUTH_SAML_SP_ENTITY_ID=
-SOCIAL_AUTH_SAML_SP_PUBLIC_CERT=
-SOCIAL_AUTH_SAML_SP_PRIVATE_KEY=
-SOCIAL_AUTH_SAML_ORG_DISPLAYNAME=
-SOCIAL_AUTH_SAML_CONTACT_NAME=
-SOCIAL_AUTH_SAML_SECURITY_ENCRYPTED=True
-
-# SAML IDP
-SOCIAL_AUTH_SAML_IDP_ENTITY_ID=
-SOCIAL_AUTH_SAML_IDP_URL=
-SOCIAL_AUTH_SAML_IDP_ATTRIBUTE_PERM_ID=
-SOCIAL_AUTH_SAML_IDP_ATTRIBUTE_NAME=
-SOCIAL_AUTH_SAML_IDP_ATTRIBUTE_EMAIL=
-SOCIAL_AUTH_SAML_IDP_X509=
-
-USE_X_FORWARDED_PORT=
-USE_X_FORWARDED_HOST=
```

Comment:
Will the application start without these forwarded settings?

Reply:
It certainly does when running locally. In production environments it might not, which is why I made the comment above asking if the `.env.example` file has any special importance to devops.